README.md

```diff
@@ -84,17 +86,17 @@ We currently support SQS subscriptions only, but keep checking back for other me
 * This is where you pass on to your BLL layer.
 * We also need to tell the stack whether we handled the message as expected. We can say things got messy either by returning false, or bubbling up exceptions.
 
-````c#
-        public class OrderNotifier : IHandler<OrderAccepted>
-        {
-            public bool Handle(OrderAccepted message)
-            {
-                // Some logic here ...
-                // e.g. bll.NotifyRestaurantAboutOrder(message.OrderId);
-                return true;
-            }
-        }
-````
+```csharp
+public class OrderNotifier : IHandler<OrderAccepted>
+{
+    public bool Handle(OrderAccepted message)

```

Comment:
I would prefer the first example to use the async version, as the sync `IHandler` is deprecated.